docs/guides/admin/docs/modules/player.configuration.md

```diff
@@ -2,32 +2,81 @@ Opencast Player - Configuration
 ===============================
 
 The configurations for the player are done for each tenant. So the configuration keys are located in
-`.../etc/org.opencastproject.organization-mh_default_org.cfg`.
+`.../etc/ui-config/<tenant>/theodul/config.yml`
+
+The default tenant for opencast is `mh_default_org`
```

Comment:
This line is superfluous since wither you have multi-tenancy configured and you know about tenant configurations and identifiers or you don't in which case there is just a single tenant and it becomes obvious.

Reply:
Obvious because there is a folder `mh_default_org` with files in it? I don't know if this name alone is clear or confusing. I don't see the harm in making it explicit here.

Reply:
Yep, that is why I put explicit because is someone changes the tenant, maybe is not aware that also this has to be changed.